packages/yoroi-extension/app/containers/settings/categories/WalletSettingsPage.js

```diff
@@ -70,8 +72,9 @@ export default class WalletSettingsPage extends Component <StoresProps> {
           activeField={walletFieldBeingEdited}
           nameValidator={name => isValidWalletName(name)}
         />
+        <NotificationsSettings intl={intl}/>
```

Comment:
This needs to be feature-flagged for now, if we want to be able to merge to develop

Reply:
Perfect, adding the FF